package.json

```diff
@@ -107,6 +107,7 @@
     "test": "node --max-old-space-size=4096 --trace-deprecation node_modules/jest-cli/bin/jest",
     "test:update-snapshots": "yarn jest -u",
     "test:integration": "node --max-old-space-size=4096 --trace-deprecation node_modules/jest-cli/bin/jest --testMatch \"<rootDir>/test/*.test.js\"",
+    "test:HMR": "node --max-old-space-size=4096 --trace-deprecation node_modules/jest-cli/bin/jest --testMatch \"<rootDir>/test/{HotModuleReplacementPlugin}.test.js\"",
```

Comment:
Not needed. You can run `yarn jest HotModuleReplacementPlugin`. It also runs as part of the `yarn test:integration`.

Reply:
Thank for your help